### ISSUE TITLE
Creating norm_info_mgc_lf0_vuv_bap_63_MVN.dat 

### ISSUE THREAD

**Issue by PetrochukM** [facebookresearch/loop#49]
Hi There!

Okay. So I was able to `extract_feats.py` for the full VCTK dataset. During that process, the VCTK dataset is split up into 8 temporary directories. For each of these, a `tmp_results` is created with the numpy features. Unfortunately, the `norm_info` `norm_info_mgc_lf0_vuv_bap_63_MVN` is not saved. `norm_info_mgc_lf0_vuv_bap_63_MVN` is the std and the mean of __?__. In order to combine the temporary splits and to compute the `norm_info` file, what do I need to do?

tl;dr The VCTK dataset splits only have the numpy features and do not compute a `norm_info_mgc_lf0_vuv_bap_63_MVN`; how do I edit the script to compute one that uses all the data?

**Comment by niravpatel2008** [comment]
@PetrochukM i also have large dataset for single speaker. it around 13k files. can you please share you changes so i can apply to my side.

**Comment by macarbonneau** [comment]
This post will answer your question: #11

**Comment by PetrochukM** [comment]
@macarbonneau Have you run this? Is it OKAY to just recompute the mean and the STD?

It looks like the data is recreated in the generation code using the mean and the STD; therefore, every numpy features (.npz) file needs to be updated to fit the new mean and the new STD. I want to make sure i'm understanding this correctly.

**Comment by macarbonneau** [comment]
This sounds very complicated. 

Why not just put all your .wav files in the same folder and extract npz feature on this folder. Anyway, you will have to combine them in a numpy_features/ and a numpy_features_valid/ folder later. Voiceloop uses the prefix in your file name to determine speaker ID. 

This is what I did and it works fine.

**Comment by PetrochukM** [comment]
They are in the same folder. 

When I run `extract_feats` with 40,000 files, it splits them up because its over a 5,000 file limit here:
https://gist.github.com/kastnerkyle/cc0ac48d34860c5bb3f9112f4d9a0300#file-extract_feats-py-L1339

Did you disable this feature?

**Comment by macarbonneau** [comment]
Oh yeah...    I played a bit with the script. I don't think this part is active anymore in my code.

**Comment by PetrochukM** [comment]
@macarbonneau O nice. What script did you use? How do I generate what I need?
